pkg/client/core/report/console/container_repository_console.go

```diff
@@ -0,0 +1,41 @@
+package console
```

Comment:
You can keep it if you want to, but I will remove it in my state PR, so if you waaaant to 😬